.github/workflows/ci_cron_daily.yml

```diff
@@ -57,19 +58,20 @@ jobs:
 
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         operations-per-run: 50
-        ascending: true
+        ascending: false
 
         days-before-issue-stale: -1
         days-before-issue-close: 7
         remove-issue-stale-when-updated: false
         close-issue-message: 'I am going to close this issue because it was marked with **Close?**, but if you feel that this issue should stay open, then feel free to re-open, remove **Close?** and **closed-by-bot** labels, and apply **keep-open** label.'
+        stale-issue-label: 'Close?'
```

Comment:
Seems like we still need this set. Otherwise, I see something like this for #11011 , which should have been closed if this were not a dry run but instead it is saying, "This issue hasn't a stale label."
```
2021-05-20T03:18:25.6225301Z [#11011] Found this issue last updated at: 2020-11-25T03:58:26Z
2021-05-20T03:18:25.6226383Z [#11011] The option only-labels (​https://github.com/actions/stale#only-labels​) was not specified
2021-05-20T03:18:25.6227424Z [#11011] └── Continuing the process for this issue
2021-05-20T03:18:25.6228463Z [#11011] Days before issue stale: -1
2021-05-20T03:18:25.6229217Z [#11011] The issue is not closed nor locked. Trying to remove the close label...
2021-05-20T03:18:25.6230114Z [#11011] This issue hasn't a stale label
2021-05-20T03:18:25.6231457Z [#11011] The option any-of-labels (​https://github.com/actions/stale#any-of-labels​) was specified to only process the issues and pull requests with one of those labels (1)
2021-05-20T03:18:25.6232768Z [#11011] ├── One of the required labels is present on this issue
2021-05-20T03:18:25.6233670Z [#11011] └── Continuing the process for this issue
2021-05-20T03:18:25.6234284Z [#11011] This issue has no milestone
2021-05-20T03:18:25.6235010Z [#11011] └── Skip the milestones checks
2021-05-20T03:18:25.6243632Z [#11011] This issue has no assignee
2021-05-20T03:18:25.6244572Z [#11011] └── Skip the assignees checks
2021-05-20T03:18:25.6245232Z [#11011] This issue is not stale
2021-05-20T03:18:25.6246363Z [#11011] This issue should be stale based on the last update date the 25-11-2020 (2020-11-25T03:58:26Z)
2021-05-20T03:18:25.6248158Z [#11011] This issue should not be marked as stale based on the option days-before-issue-stale (​https://github.com/actions/stale#days-before-issue-stale​) (-1)
```